bazel/google_cloud_cpp_deps.bzl

```diff
@@ -75,10 +75,10 @@ def google_cloud_cpp_deps():
         http_archive(
             name = "com_google_googleapis",
             urls = [
-                "https://github.com/googleapis/googleapis/archive/59f97e6044a1275f83427ab7962a154c00d915b5.tar.gz",
+                "https://github.com/googleapis/googleapis/archive/spanner_commit_stats.tar.gz",
             ],
-            strip_prefix = "googleapis-59f97e6044a1275f83427ab7962a154c00d915b5",
-            sha256 = "5e785c25b1d57973e7481b4da226d7c73056ea22c7545bf6d14dbebf6e99b073",
+            strip_prefix = "googleapis-spanner_commit_stats",
+            sha256 = "198b878c673acbce48335b6de7575ff658106d75bb1b0616a0e0f27402a90356",
```

Comment:
FWIW, the SHA256 will change if they ever push a new commit to the `spanner_commit_stats` branch (or they rebase the branch, or otherwise change its contents). This would break the build for anybody who has downloaded the "preview". Not sure how likely it is that new commits appear on that branch.

I think you can omit the `sha256` setting, at a risk of builds failing because the downloaded tarball is bad (git can return something that is not a tarball if its servers are in trouble). I think Bazel does better caching if the `sha256` setting is there.

Reply:
Acknowledged.  I think (at least for now) given the benefits and that a change is unlikely, that I'll leave as-is.  If I end up being wrong I will happily accept an "I told you so".  Thanks.